console/src/main/scripts/plugins/accounts/ts/accountsPlugin.ts

```diff
@@ -17,87 +17,69 @@
 
 /// <reference path='accountsGlobals.ts'/>
 module HawkularAccounts {
-    export var _module = angular.module(HawkularAccounts.pluginName, ['ui.bootstrap']);
-    var accountsTab:any = undefined;
```

Comment:
Since we doing ES6 you do a global replace of 'var' with 'let' (and even const) to eliminate the accidental bugs that var allows.